osx-setup/distribution.xml

```diff
@@ -10,7 +10,7 @@
   <conclusion file="conclusion.html" mime-type="text/html" />
   <!-- List all component packages -->
   <pkg-ref id="network.loki.lokinet"
-           version="0.4"
+           version="0.6"
```

Comment:
0.5?

Reply:
@majestrate this PR is to be merged no earlier than tomorrow, once the v0.5 release is out?